src/map/mob.cpp

```diff
@@ -3616,7 +3614,7 @@ int32 mob_dead(struct mob_data *md, struct block_list *src, int32 type)
 	if( md->can_summon )
 		mob_deleteslave(md);
 
-	map_freeblock_unlock();
+	freeLock.unlock();
```

Comment:
Unrelated to this PR, but isnt this too early here?
"md" is still used below here and may be freed by calling unlock here.